### DISCUSSION TITLE
Styling a generic template does not seem to work?

### DISCUSSION THREAD

**Discussion by tyeeman** [datenstrom/community#383] [Ask a question] [answered]
I'm trying to covert a generic template to work in yellow but having a slow time. I got the content to populate but trying to get the top nav working is not. When I use the snippet - 
```
<?php $yellow->snippet($yellow->page->get("navigation")) ?>
```
I see a list with bulleted items and my template shows it vertically instead of horizontally. My generic template only uses anchors not UL, so does that mean this will not work? 

One more question. Do I need any of this in my generic template - 
```
<?php $yellow->page->set("pageClass", "page") ?>
<?php $yellow->page->set("pageClass", $yellow->page->get("pageClass")." template-".$yellow->page->get("template")) ?>
<?php if($yellow->page->get("navigation")=="navigation-sidebar") $yellow->page->setPage("sidebar", $yellow->page); ?>
<?php if($page = $yellow->pages->find($yellow->lookup->getDirectoryLocation($yellow->page->location).$yellow->page->get("sidebar"))) $yellow->page->setPage("sidebar", $page) ?>
<?php if($yellow->page->isPage("sidebar")) $yellow->page->set("pageClass", $yellow->page->get("pageClass")." with-sidebar") ?>
<div class="<?php echo $yellow->page->getHtml("pageClass") ?>">
```
The above is in the body of the "header.php" file.

So all I need are these two tags to work and I'm set - 
```
<?php $yellow->snippet("sidebar") ?>
<?php $yellow->snippet($yellow->page->get("navigation")) ?>
```
I haven't started making the blog portion work though! I hope that's easier.

**Comment by schulle4u** [comment]
Yellow uses some CSS to style the page output. Just copying the necessary php code therefore won't work out of the box. As a starting point I suggest to have a look at [the default style sheet](https://github.com/datenstrom/yellow-themes/blob/master/flatsite/flatsite.css), and adapt it for your template.

**Comment by tyeeman** [comment, marked as answer]
Well, this fixed it - thanks to w3schools.com
```
ul {
  list-style-type: none;
  margin: 0;
  padding: 0;
}

li {
  display: inline;
}
```
